src/localsocketcontroller.h

```diff
@@ -44,8 +50,19 @@ class LocalSocketController final : public ControllerImpl {
   void errorOccurred(QLocalSocket::LocalSocketError socketError);
   void readData();
   void parseCommand(const QByteArray& command);
-
-  void write(const QJsonObject& json);
+  void disarmTimeout(const QString& responseType);
+  void clearAllTimeouts();
+
+  /**
+   * @brief Write a JSON message to the socket, sending it to the daemon.
+   *
```

Comment:
Are repsonseType or expectedResponseType better names? And should json be renamed to something like message or request or command?

Reply:
I have updated the names accordingly.